draft-ietf-rtcweb-jsep.xml

```diff
@@ -1618,10 +1669,10 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           </t>
 
           <t>The next step is to generate session-level lip sync groups as
-          defined in <xref target="RFC5888"/>, Section 7.  For each MediaStream
-          with more than one MediaStreamTrack, a group of type "LS" MUST be
-          added that contains the mid values for each MediaStreamTrack in that
-          MediaStream. </t>
+          defined in <xref target="RFC5888"/>, Section 7. For each MediaStream
+          with more than one associated RtpTransceiver, a group of type "LS"
```

Comment:
This text seems odd; do MediaStreams have associated transceivers? Maybe say 'for each MediaStream referenced by more than one transceiver'.